mkdocs.yml

```diff
@@ -167,7 +167,9 @@ nav:
       - Permissioning:
         - Create a permissioned network: Tutorials/Permissioning/Create-Permissioned-Network.md
         - Get started with onchain permissioning: Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md
-        - Upgrade the permissioning contract: Tutorials/Permissioning/Upgrade-Permissioning-Contract.md
+        - Upgrading permissioning contracts:
+          - Version <2.0: Tutorials/Permissioning/Upgrading Permissioning Contracts/Upgrade-Permissioning-Contract-v1.md
```

Comment:
why not "1.x" instead of "<2.0" it's a bit confusing as it's too close visually to "2.0+"
Then "Version 1" and "Version 2" would be enough imo. Plus it would match the page path (better SEO).

Reply:
Be careful, could you please replace the space with a `-` in the folder name?
`Upgrading Permissioning Contracts` -> `Upgrading-Permissioning-Contracts`